src/xy/xyEnsureGrowingX.js

```diff
@@ -0,0 +1,14 @@
+export function xyEnsureGrowingX(x, y) {
```

Comment:
Please add jsdoc for this method and use the same API as all the other methods (it should be an object)
You can also check if the format is correct using a simple method
https://github.com/mljs/spectra-processing/blob/6331061f87e5358b37332b0843403b4979132696/src/xy/xyJoinX.js#L11-L13